crates/viewer/re_space_view_graph/src/ui/draw.rs

```diff
@@ -281,29 +283,58 @@ pub fn draw_graph(
         let response = match node {
             Node::Explicit { instance, .. } => {
                 let highlight = entity_highlights.index_highlight(instance.instance_index);
-                let response = draw_node(ui, center, node.label(), highlight);
-
-                let response = if let Label::Text { text, .. } = &instance.label {
-                    response.on_hover_text(format!(
-                        "Graph Node: {}\nLabel: {text}",
-                        instance.graph_node.as_str(),
-                    ))
-                } else {
-                    response.on_hover_text(format!("Graph Node: {}", instance.graph_node.as_str(),))
-                };
+                let mut response = draw_node(ui, center, node.label(), highlight);
 
                 let instance_path =
                     InstancePath::instance(entity_path.clone(), instance.instance_index);
                 ctx.select_hovered_on_click(
                     &response,
-                    vec![(Item::DataResult(query.space_view_id, instance_path), None)].into_iter(),
+                    vec![(
+                        Item::DataResult(query.space_view_id, instance_path.clone()),
+                        None,
+                    )]
+                    .into_iter(),
```

Comment:
You should be able to pass a single item here.

```suggestion
                    Item::DataResult(query.space_view_id, instance_path.clone()),
```